qa/L0_trace/test.sh

```diff
@@ -691,6 +691,132 @@ if [ `grep -o 'parent_id' bls_trace.log | wc -l` != "2" ]; then
     RET=1
 fi
 
+# Attempt to trace non-existent model
+SERVER_ARGS="--trace-file=trace_off_to_min.log --trace-level=OFF --trace-rate=1 --model-control-mode=explicit --model-repository=$MODELSDIR"
```

Comment:
1. Use `--trace-config` as the other trace args are deprecated in favor of trace-config.
2. Don't set trace-level / trace-rate if they're not relevant to the test, just use defaults to keep server args minimal to what you need

Reply:
example of trace config is in the test closer to the end: https://github.com/triton-inference-server/server/pull/6199/files#diff-27e9ca4cd8f292f0e5a94bbabb243e40fe0d47cd55164a0cb49ae6e093c9fbe7R840

Since some of these tests were before we switched to `trace-config`, not all of them were adjusted